test/CMakeLists.txt

```diff
@@ -1,93 +1,113 @@
 message(STATUS "Building Unit Tests ${UNITTEST}")
 
+# look for Python3
+find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
```

Comment:
This one, and the resulting `Python3_EXECUTABLE` variable, is really helpful! :+1: